client/client.go

```diff
@@ -521,13 +508,14 @@ func (ec *SDKClient) TraceBlockByHash(
 
 	var calls []*rpcCall
 	var raw json.RawMessage
+
 	err := ec.CallContext(ctx, &raw, "debug_traceBlockByHash", blockHash, ec.tc)
 
+
```

Comment:
I think one line is enough between two statements but generally speaking I think you can avoid both lines since you are checking error for the above statement